modules/openapi-generator/src/main/resources/JavaPlayFramework/module.mustache

```diff
@@ -11,5 +12,6 @@ public class Module extends AbstractModule {
         bind({{classname}}ControllerImpInterface.class).to({{classname}}ControllerImp.class);
         {{/apis}}
         {{/apiInfo}}
+		bind(SecurityAPIUtils.class);
```

Comment:
please fix the indentation here

Reply:
I do not know why, but the indentation in generated code is correct. So if I change it here, it will no longer be correct in the generated code.

Reply:
it is not ok in the generated code, see my comment below

Reply:
ok, there was "tab" instead of "spaces". Will be fixed in next push.